CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
```

Comment:
```suggestion
## Unreleased
```

Reply:
We're adding the unreleased or development tag with the first pr, so let's stick with that.

Reply:
Looks like your suggestion doesn't change any code, I'm a little bit confusing what you are asking.

Reply:
It just seems strange to me to leave the unreleased tag empty; it would be better to remove it and add it with the first pull request.

Reply:
OK